ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -1030,17 +1024,17 @@ - (void)updatePendingAndFinishedPromotions:
   });
 }
 
-- (void)fetchPromotions:(nullable void (^)(NSArray<LedgerPromotion*>* grants,
-                                           bool shouldReconcileAds))completion {
+- (void)fetchPromotions:
+    (nullable void (^)(NSArray<LedgerPromotion*>* grants))completion {
   ledger->FetchPromotions(
       ^(ledger::type::Result result,
         std::vector<ledger::type::PromotionPtr> promotions) {
         if (result != ledger::type::Result::LEDGER_OK) {
           return;
         }
-        [self updatePendingAndFinishedPromotions:^(bool shouldReconcileAds) {
+        [self updatePendingAndFinishedPromotions:^() {
```

Comment:
`()` can be omitted here to just `^{` but I think it will still compile

Reply:
Fixed